dace/sdfg/sdfg.py

```diff
@@ -298,6 +304,19 @@ class SDFG(OrderedDiGraph[SDFGState, InterstateEdge]):
 
     debuginfo = DebugInfoProperty(allow_none=True)
 
+    _pgrids = Property(dtype=dict,
+                       desc="Process-grid descriptors for this SDFG",
+                       to_json=_arrays_to_json,
+                       from_json=_arrays_from_json)
+    _subarrays = Property(dtype=dict,
+                          desc="Sub-array descriptors for this SDFG",
+                          to_json=_arrays_to_json,
+                          from_json=_arrays_from_json)
+    _rdistrarrays = Property(dtype=dict,
+                             desc="Sub-array redistribution descriptors for this SDFG",
+                             to_json=_arrays_to_json,
+                             from_json=_arrays_from_json)
+
```

Comment:
I am not a fan of spamming properties in SDFG. We have to find somewhere else.

Reply:
There are properties of the (DaCe) program though.